### ISSUE TITLE
causes client side crash 

### ISSUE THREAD

**Issue by 3dguru** [ShetiPhian/Localizations-and-Issues#379]
When launching Minecraft it crashes when shetiphiancore

**Additional context**
Minecraft Version:  1.19.2
Fabric Version: 0.14.10
ShetiPhian Core Version: 1.3.2
My Mod(s) Name & Version: 
Mods:
  [✔️] [Universal]CTOV-v3-1-2
  [✔️] 3dskinlayers-fabric-1.5.2-mc1.19.1
  [✔️] AdditionalAdditions-5.1.0
  [✔️] AdditionalStructures-1.19.x-(v.4.1.0)
  [✔️] Adorn-3.7.0+1.19.2-fabric
  [✔️] adventurez-1.4.17
  [✔️] amethyst_core-0.4.0+1.19
  [✔️] amethyst_imbuement-1.19-17
  [✔️] animatica-0.5+1.19
  [✔️] another_furniture-fabric-2.1.2-1.19.2
  [✔️] appleskin-fabric-mc1.19-2.4.1
  [✔️] architectury-6.3.49-fabric
  [✔️] ArmorStatues-v4.0.2-1.19.2-Fabric
  [✔️] AttributeFix-Fabric-1.19.2-17.1.2
  [✔️] auto_third_person-fabric-1.19.2-1.2.1
  [✔️] BarteringStation-v4.0.3-1.19.2-Fabric
  [✔️] BetterAdvancements-Fabric-1.19.2-0.2.2.142
  [✔️] betteranimalsplus-1.19-11.0.7-fabric
  [✔️] BetterAnimationsCollection-v4.0.4-1.19.2-Fabric
  [✔️] bettercombat-fabric-1.4.5+1.19
  [✔️] BetterF3-4.0.0-Fabric-1.19.2
  [✔️] betterfpsdist-fabric-1.19.1-1.9
  [✔️] betterstats-1.4.3
  [✔️] BetterThirdPerson-Fabric-1.19-1.8.1
  [✔️] bettervillage-fabric-1.19.2-2.0.0
  [✔️] biomemakeover-FABRIC-1.19.2-1.5.12
  [✔️] BiomeParticleWeather-v4.0.0-1.19-Fabric
  [✔️] blur-2.6.0
  [✔️] BOMD-1.4.3-1.19.2
  [✔️] Bookshelf-Fabric-1.19.2-16.1.5
  [✔️] cardinal-components-api-5.0.2
  [✔️] carrier-1.11.0
  [✔️] cem-0.7.1
  [✔️] charm-fabric-1.19.2-4.4.4
  [✔️] charmofundying-fabric-6.0.0+1.19.2
  [✔️] chat_heads-0.7.1-fabric-1.19.1
  [✔️] cherishedworlds-fabric-6.0.1+1.19.2
  [✔️] chipped-1.2+1.19
  [✔️] CITResewn-1.1.2+1.19.2
  [✔️] cloth-api-4.0.65
  [✔️] cloth-config-8.2.88-fabric
  [✔️] collectors-album-1.19.2-1.4.0
  [✔️] colormatic-3.1.2+mc.1.19
  [✔️] combatroll-fabric-1.0.8+1.19
  [✔️] comforts-fabric-6.0.2+1.19.2
  [✔️] conjuring-1.0.25+1.19
  [✔️] continuity-2.0.1+1.19
  [✔️] Controlling-fabric-1.19.2-10.0+7
  [✔️] CorgiLib-fabric-1.19.2-1.0.0.6
  [✔️] crawl-0.11.1
  [✔️] CreativeCore_FABRIC_v2.8.5_mc1.19.2
  [✔️] Croptopia-1.19.2-FABRIC-2.1.2
  [✔️] CullLessLeaves-1.0.6
  [✔️] dark-loading-screen-1.6.12
  [✔️] darkloot-1.0.7
  [✔️] darkquesting-1.0.7
  [✔️] dashloader-4.1.3+1.19
  [✔️] Debugify-2.8.0
  [✔️] Decorative Blocks-fabric-1.19.2-3.0.0
  [✔️] deepslatecutting-fabric-1.5.0
  [✔️] dontdropit-2.5.0+1.19
  [✔️] dynamiccrosshair-3.11+1.19.2
  [✔️] dynamiccrosshair-compat-1.11+1.19.2
  [✔️] dynamic-fps-2.2.0
  [✔️] earthtojavamobs-fabric-1.9.3+1.19.2
  [✔️] EasyAnvils-v4.0.5-1.19.2-Fabric
  [✔️] ecologics-fabric-1.19.2-2.1.10
  [✔️] effective-1.3
  [✔️] eldritch-mobs-1.12.7
  [✔️] elytraslot-fabric-6.0.0+1.19.2
  [✔️] EmeraldTools-Fabric-1.19.2-1.4.38
  [✔️] EnchantmentDescriptions-Fabric-1.19.2-13.0.3
  [✔️] Enhanced_Celestials-fabric-1.19.2-2.1.0.2
  [✔️] EnhancedVisuals_FABRIC_v1.5.8_mc1.19.1
  [✔️] entity_texture_features_fabric_1.19.x-4.1.1
  [✔️] entityculling-fabric-1.5.2-mc1.19
  [✔️] everycomp-1.19.2-2.1.0
  [✔️] expandeddelight-0.2.2
  [✔️] extraalchemy-fabric-1.19.2-1.9.0
  [✔️] fabric-api-0.64.0+1.19.2
  [✔️] fabric-end-remastered-5.2.0
  [✔️] fabric-language-kotlin-1.8.5+kotlin.1.7.20
  [✔️] farmers-delight-fabric-1.19.X-1.3.5
  [✔️] farmersknives-2.2
  [✔️] farsight-fabric-1.19-2.1
  [✔️] ferritecore-5.0.0-fabric
  [✔️] flan-1.19-1.8.1.1-fabric
  [✔️] ForgeConfigAPIPort-v4.2.6-1.19.2-Fabric
  [✔️] FpsReducer2-fabric-1.19.2-2.1
  [✔️] frame-0.26.1+1.19-fabric
  [✔️] friendsandfoes-fabric-mc1.19.2-1.5.12
  [✔️] geckolib-fabric-1.19-3.1.34
  [✔️] geckolib-fabric-1.19-3.1.35
  [✔️] HorseExpert-v4.0.0-1.19.2-Fabric
  [✔️] illuminations-1.10.8
  [✔️] indium-1.0.9+mc1.19.2
  [✔️] interactic-0.1.11+1.19
  [✔️] InventoryHUD-fabric-[1.19.1]+-3.4.2
  [✔️] iris-mc1.19.2-1.4.0
  [✔️] jamlib-0.4.4
  [✔️] journeymap-1.19.1-5.9.0beta2-fabric
  [✔️] krypton-0.2.1
  [✔️] lambdynamiclights-2.1.2+1.19
  [✔️] lazydfu-0.1.3
  [✔️] LeavesBeGone-v4.0.0-1.19.2-Fabric
  [✔️] libraryferret-fabric-1.19.2-3.0.0
  [✔️] lithium-fabric-mc1.19.2-0.10.1
  [✔️] mavm-1.1.0-mc1.19
  [✔️] mcda-2.2.0
  [✔️] mcdar-1.5.2-1.19
  [✔️] mcdw-5.0.4-1.19
  [✔️] memoryleakfix-1.19.1-0.7.0
  [✔️] midnightlib-fabric-1.0.0
  [✔️] minegate-moreblocks-1.2.5
  [✔️] modmenu-4.0.6
  [✔️] moonlight-1.19.2-2.1.4-fabric
  [✔️] morefrogs-1.19.x-1.2.3a-fabric
  [✔️] morevillagers-fabric-1.19-4.0.2
  [✔️] MouseTweaks-fabric-mc1.19-2.22
  [✔️] mousewheelie-1.10.2+mc1.19-pre1
  [✔️] multibeds-fabric-1.19-1.3.1
  [✔️] musketmod-1.19.2-fabric-1.4.6
  [✔️] mvs-2.5.9
  [✔️] mythicmetals-0.16.3+1.19.2
  [✔️] naturalist-fabric-2.1.1-1.19.2
  [✔️] NoChatReports-FABRIC-1.19.2-v1.13.11
  [✔️] Oh_The_Biomes_You&#039;ll_Go-fabric-1.19.2-2.0.0.13
  [✔️] optigui-1.1.6
  [✔️] owo-lib-0.8.5+1.19
  [✔️] packet_size_doubler-1.2.0
  [✔️] Patchouli-1.19.2-76-FABRIC
  [✔️] Pehkui-3.6.0+1.14.4-1.19.3
  [✔️] physics-mod-2.8.13a-mc-1.19.x-fabric
  [✔️] player-animation-lib-fabric-0.3.5
  [✔️] polymorph-fabric-0.45.0+1.19.2
  [✔️] PresenceFootsteps-1.6.3
  [✔️] probablychests-0.5.5-1.19.2
  [✔️] PuzzlesLib-v4.3.12-1.19.2-Fabric (1)
  [✔️] quickshulker-1.3.9-1.19
  [✔️] reeses_sodium_options-1.4.7+mc1.19.2-build.59
  [✔️] reinforced-barrels-2.3.0+1.19
  [✔️] reinforced-shulker-boxes-2.2.0+1.19
  [✔️] repurposed_structures_fabric-6.1.2+1.19
  [✔️] right-click-harvest-3.0.4
  [✔️] rottencreatures-fabric-1.19.2-1.0.1
  [✔️] RoughlyEnoughItems-9.1.565
  [✔️] rpgdifficulty-1.3.0
  [✔️] rpgstats-4.4.1+1.19.2
  [✔️] servercore-1.3.3-1.19.2
  [✔️] shetiphiancore-fabric-1.19-1.3.2
  [✔️] simplyswords-fabric-1.33-1.19.x
  [✔️] smoothboot-fabric-1.19-1.7.1
  [✔️] sodium-extra-0.4.11+mc1.19.2-build.68
  [✔️] sodium-fabric-mc1.19.2-0.4.4+build.18
  [✔️] soulslike-weaponry-0.9.6
  [✔️] soundphysics-fabric-1.19.2-1.0.16
  [✔️] spiderstpo-fabric-1.19.2-2.0.2
  [✔️] starlight-1.1.1+fabric.ae22326
  [✔️] Structory-1.19-v1.3
  [✔️] supplementaries-1.19.2-2.2.15-fabric
  [✔️] takesapillage-1.19-1.0.2
  [✔️] TerraBlender-fabric-1.19.2-2.0.1.129
  [✔️] Terralith_v2.3.5
  [✔️] The_Graveyard_2.0_(FABRIC)_for_1.19.2
  [✔️] The_Graveyard_Biomes_1.2_(FABRIC)_for_1.19
  [✔️] things-0.2.20+1.19
  [✔️] toms_storage_fabric-1.19-1.4.3
  [✔️] ToolStats-Fabric-1.19.2-12.0.2
  [✔️] transparent-5.1.2+1.19-fabric
  [✔️] travelers-backpack-1.19.2-8.2.6
  [✔️] trinkets-3.4.0
  [✔️] twigs-2.0.0-beta.9+1.19-fabric
  [✔️] visuality-0.5.3
  [✔️] VisualWorkbench-v4.2.0-1.19.2-Fabric
  [✔️] voicechat-fabric-1.19.2-2.3.13
  [✔️] WilderWild-1.1.1-Fabric+1.19.2
  [✔️] wildlife-1.19.2-2.2
  [✔️] winged-1.19-3.3.0
  [✔️] wolves-with-armor-1.8.0-1.19
  [✔️] worldedit-mod-7.2.12
  [✔️] wraith-alloy-forgery-2.0.16+1.19
  [✔️] wraith-waystones-3.0.5+mc1.19.2
  [✔️] wwoo-3.1.8-FABRIC-1.19.2
  [✔️] YetAnotherConfigLib-1.6.0
  [✔️] youre-in-grave-danger-1.4.5
  [✔️] YungsApi-1.19.2-Fabric-3.8.2
  [✔️] YungsBetterDesertTemples-1.19.2-Fabric-2.2.1
  [✔️] YungsBetterDungeons-1.19.2-Fabric-3.2.1
  [✔️] YungsBetterMineshafts-1.19.2-Fabric-3.2.0
  [✔️] YungsBetterOceanMonuments-1.19.2-Fabric-2.1.0
  [✔️] YungsBetterStrongholds-1.19.2-Fabric-3.2.0
  [✔️] Zoomify-2.8.0
Are other mods present? Yes
If yes:
- Does everything work without my mods? Yes
- Have you tested with only my mods? Yes

PasteBin or GitHub Gists links
latest.log = 
debug.log =
[X1E3N[1].TXT](https://github.com/ShetiPhian/Localizations-and-Issues/files/9956678/X1E3N.1.TXT)

**Comment by ShetiPhian** [comment]
My mixin is conflicting with one from debugify

The odd thig is the mixin loader claims its skipping mine due to the conflict, which would be fine, but it tries loading it anyway and since the class was changed by debugify my target no longer exists and causes a crash.

While I do have configs to disable my mixins, those only stop the mixin form making changes, the code is still injected.

I should just remove that mixin completely, its been a hassle since it was added due to so many other mods doing it also. I thought I finally found a problem fee way via redirect, but turns out I was wrong. Honestly Mojang should be supporting line breaks in their tooltips from the get go. As it is Fabric should do as Forge did and make it part of their base changes so mods don't to add it and cause conflicts.
